### PR TITLE
refactor: Renamed "tip" to "contribute" in the player's channel view.

### LINKED ISSUE #106: Rename "tip" to "contribute"
**What's the problem? (required)**

The concept of "tipping" isn't really common parlance outside North America. It's maybe the kind of thing that 

**What's the current behavior? (required)**

The text currently reads as follow:

image

**Do you have any personal experiences dealing with this problem? (optional)**

People have asked me "what does it mean 'tip'"?

**Describe a possible solution you've considered (optional)**

Text could say: "Contribute directly to the broadcaster [<3 CONTRIBUTE]"

**Additional context (optional)**

If the creators and broadcasters are always considered as people to "tip", it might make them feel less valued compared to other members of community who might be "paid" or "funded" or "bountied". Just a thought.

### PR DESCRIPTION
**Pull request by saskakol**
Changed some copy to change "tip" to "contribute" on the player's channel page.

fix #106

**What does this pull request do? Explain your changes. (required)**
This simply changes some of the copy on the channel page in the player package to replace the word "tip" with "contribute".

**Specific updates (required)**
- Changed copy in packages/player/src/views/Channel/index.js

**How did you test each of these updates (required)**
No automated testing, since it's just a copy change.

**Does this pull request close any open issues?**
Fixes #106 

**Checklist:**
- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/player/src/views/Channel/index.js [hunk 1]
```diff
@@ -295,14 +295,14 @@ class Channel extends Component {
           onClose={() => this.closeModal('tip')}
           center
         >
-          <h2>Leave a tip</h2>
+          <h2>Contribute directly to a broadcaster</h2>
```
Comment (Randy1Burrell) [inline packages/player/src/views/Channel/index.js]: Being that we are referring to the current broadcaster you are viewing, shouldn't this be "Contribute directly to this broadcaster"?

Comment (saskakol) [reply packages/player/src/views/Channel/index.js]: You’re right, I’ll make the change.

**Comment by saskakol** [comment]
@Randy1Burrell Fixed in the new commit.

**Review by Randy1Burrell — APPROVED** [review]
Great job @iSasFTW!!

**Comment by Randy1Burrell** [comment]
@iSasFTW thanks. Great job!!

**Pull request merged**